scripts/rebuild-lesson-plan-06.ts

```diff
@@ -3,11 +3,6 @@ import * as getStdin from "get-stdin";
 import * as RegexEscape from "regex-escape";
 
 
```

Comment:
This file is what I used to do most of the reformatting of the lesson plan into something we can use.  Hopefully most of it is reusable between the different lesson plans.